### ISSUE TITLE
again hard (show answer button problem)

### ISSUE THREAD

**Issue by noriul** [ankitest/anki-musthave-addonz-by-ankitest#1]
is there any way to change the show answer button to be glued to the bottom of the screen? When i'm using it on my tablet i can never seem to hit it correctly and it's been driving me crazy these past few days. The [   :(   :)   ]buttons work very well i wish to find the way to change the show answer button to be the same as them.

**Comment by ankitest** [comment]
Sorry, I can't understand, what's wrong with _Show Answer_ button 4U? It is located exactly on the place of  [ :( :) ] buttons. Can you create a screenshot of your problem?

**Comment by noriul1** [comment]
Firstly i wanted to apologize for taking so much time to reply. 

> Sorry, I can't understand, what's wrong with Show Answer button 4U?

Nothing is actually wrong with the Show Anwer button. My problem stems from another place entirely. You see as i mention before, I use Anki primarily on my small cheap windows tablet. However, the touch calibration driver(or whatever is the original name) is pretty terrible( and trust me i tried nearly everything short of making my own to fix it) and for example if i want to press something that is located near the edges of the screen, the touch driver anchors the click at the **very** end of the screen. So from here the problem originates.
IF you look at the following picture 
![showanswerbutton](https://cloud.githubusercontent.com/assets/26793442/24504850/120170e4-1558-11e7-8c7b-e8ec2c382145.png)
you would notice some of the empty space i outlined in orange. 
So lets say i press somewhere around the middle height of the ShowAnswer button the click will register at the very edge a.k.a **at the bottom of the Orange Empty space**. 
Therefore whenever i want to see the back card in Anki i would need to press the button 2-3 times till i find that sweet spot that will allow me to proceed. It's really really **really** frustrating, and if you do it enough times it will drive you up against the wall.  : ( .

![badgoodbuttons](https://cloud.githubusercontent.com/assets/26793442/24505517/75ef2b1c-155a-11e7-97c0-4a1508e482d0.png)
Here you can see that there is next to no space so even if you click at the very end of the screen it will still register as clicking the button because its "glued" to the edge.


To summarize the big question was this : **What part of the addon i would need to edit and how, that will result in "gluing" the ShowAnswerButton just like SmileyFaceButtons** ?

P.S. In case you didn't notice i use Anki with the ToggleFullscreen addon. Thanks for hearing me.

**Comment by ankitest** [comment]
I can't do anything with this Supplement.
You can try 
[spec4U edition,](https://github.com/ankitest/anki-musthave-addonz-by-ankitest/blob/master/_Again_Good_wide_big_buttons.py)
Maybe this one will suite you?

**Comment by noriul1** [comment]
Yup those work i used them earlier before switching.

**Comment by ankitest** [comment]
You couldn't use it before, this one was created specially for you. 
If I understand you right, everything seems to work Ok?

**Comment by noriul1** [comment]
Yes the touch  works wonderful, though is is there a way to change from again_hard to again_good?
I tried the "Low Key Anki: Pass/Fail " addon but there i have the very same touch problems just like here before you changed it.

**Comment by ankitest** [comment]
Keep attention, please.
The name of new addon is Again Good wide big buttons :-)

**Comment by noriul1** [comment]
Yes my mistake ^.^. Everything is fine now. Thanks a lot for the help I'll be sure to rate
